docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+For the first iteration of the protocol, yes. We have plans to make it cheaper and more economically attractive after mainnet with no resealing required and other perks.
+
+### What’s the minimum time period for the storage contract between the provider and the buyer?
+
+The minimum duration for a deal is set in the miner’s ask. There’s also a practical limitation, because sectors have a minimum duration (currently one month).
+
+### After I made a deal with a miner, and sent my data to them, how exactly is the data supposed to be recoverable and healable if that miner goes down?
+
+Automatic repair of faulted data is a feature we've pushed off until after mainnet launch. For now, the way to ensure resiliency is to store your data with multiple miners, to gain some level of redundancy. If you want to learn more about how we are thinking about repair in the future, [here are some notes](https://github.com/filecoin-project/specs/pull/245/files).
+
+### How do I know that whatever miner "takes the burden" would not immediately start asking me ridiculously high prices for retrieval?
+
+To avoid extortion, always ensure you store your data with a fairly decentralized set of miners (and note: it's pretty difficult for a miner to be sure they are the only person storing a particular piece of data, especially if you encrypt the data).
+
+Miners currently provide a 'dumb box' interface and will serve anyone any data they have. Maybe in the future miners will offer ACLs and logins and such, but that requires that you trust the miner. The recommended (and safest) approach here is to encrypt data you don't want others to see yourself, before storing it.
+
+### How does updating data with Filecoin work?
+
+We have some really good ideas around 'warm' storage (that is mutable, and provable) that we will probably implement in the near future. But for now, your app will have to treat Filecoin as an append-only log. If you want to change your data, you just write new data.
+
+‘Warm' storage can be done with a small amount of trust, where you make a deal with a miner with a start date quite far in the future. The miner can choose to store your data in a sector now (but they won't get paid for proving it until the actual start date) or they can hold it for you (and even send you proofs of it on request) and you can then send them new data to overwrite it, along with a new storage deal that overwrites the previous one.
+
+There’s a pretty large design space here, and we can do a bunch of different things depending on the levels of trust involved, the price sensitivity, and the frequency of updates clients desire.
+
+### What are the standards and conditions of the verifier, is there a selected institution to verify, can you tell us about this situation?
+
+There will be several different institutions. We don’t have more updates about Verfiers at this time. We will have more around mid July.
```

Comment:
We don’t have more updates about verifiers at this time. We will post on our blog once we do.